test/modules/linear_boltzmann_solvers/transport_transient/transient_transport_1d_3.lua

```diff
@@ -54,58 +49,56 @@ xs_weak_fuelB_micro = xs.Create()
 xs.Set(xs_weak_fuelB_micro, OPENSN_XSFILE, "tests/transport_transient/xs_inf_weak_1g.xs")
 
 atom_density = 0.056559
-xs_strong_fuel = xs.MakeScaled(xs_strong_fuel_micro, atom_density) --critical
-xs_weak_fuelA = xs.MakeScaled(xs_weak_fuelA_micro, atom_density) --critical
-xs_weak_fuelB = xs.MakeScaled(xs_weak_fuelB_micro, atom_density)   --critical
+xs_strong_fuel = xs.MakeScaled(xs_strong_fuel_micro, atom_density) -- critical
+xs_weak_fuelA = xs.MakeScaled(xs_weak_fuelA_micro, atom_density) -- critical
+xs_weak_fuelB = xs.MakeScaled(xs_weak_fuelB_micro, atom_density) -- critical
 
 num_groups = 1
 mat.SetProperty(materials[1], TRANSPORT_XSECTIONS, EXISTING, xs_strong_fuel)
 mat.SetProperty(materials[2], TRANSPORT_XSECTIONS, EXISTING, xs_weak_fuelA)
 
-mat.SetProperty(materials[1], ISOTROPIC_MG_SOURCE, FROM_ARRAY, { 0.0 })
-mat.SetProperty(materials[2], ISOTROPIC_MG_SOURCE, FROM_ARRAY, { 0.0 })
+mat.SetProperty(materials[1], ISOTROPIC_MG_SOURCE, FROM_ARRAY, {0.0})
+mat.SetProperty(materials[2], ISOTROPIC_MG_SOURCE, FROM_ARRAY, {0.0})
 
 function SwapXS(solver_handle, new_xs)
-    mat.SetProperty(materials[2], TRANSPORT_XSECTIONS, EXISTING, new_xs)
-    lbs.InitializeMaterials(solver_handle)
+  mat.SetProperty(materials[2], TRANSPORT_XSECTIONS, EXISTING, new_xs)
+  lbs.InitializeMaterials(solver_handle)
 end
 
---############################################### Setup Physics
+-- ########## Setup Physics ##########
 phys1 = LBSCreateTransientSolver()
 
---========== Groups
+-- ========== Groups
 grp = {}
-for g = 1, num_groups do
-    grp[g] = LBSCreateGroup(phys1)
-end
+for g = 1, num_groups do grp[g] = LBSCreateGroup(phys1) end
 
---========== ProdQuad
+-- ========== ProdQuad
 pquad = aquad.CreateProductQuadrature(GAUSS_LEGENDRE, 16)
 
---========== Groupset def
+-- ========== Groupset def
 gs0 = LBSCreateGroupset(phys1)
 cur_gs = gs0
 LBSGroupsetAddGroups(phys1, cur_gs, 0, num_groups - 1)
 LBSGroupsetSetQuadrature(phys1, cur_gs, pquad)
 LBSGroupsetSetAngleAggDiv(phys1, cur_gs, 1)
 LBSGroupsetSetGroupSubsets(phys1, cur_gs, 8)
 LBSGroupsetSetIterativeMethod(phys1, cur_gs, KRYLOV_GMRES)
---LBSGroupsetSetIterativeMethod(phys1,cur_gs,KRYLOV_RICHARDSON)
+-- LBSGroupsetSetIterativeMethod(phys1,cur_gs,KRYLOV_RICHARDSON)
 LBSGroupsetSetResidualTolerance(phys1, cur_gs, 1.0e-6)
 LBSGroupsetSetMaxIterations(phys1, cur_gs, 1000)
 LBSGroupsetSetGMRESRestartIntvl(phys1, cur_gs, 100)
---LBSGroupsetSetWGDSA(phys1,cur_gs,30,1.0e-4,false," ")
---LBSGroupsetSetTGDSA(phys1,cur_gs,30,1.0e-4,false," ")
+-- LBSGroupsetSetWGDSA(phys1,cur_gs,30,1.0e-4,false," ")
+-- LBSGroupsetSetTGDSA(phys1,cur_gs,30,1.0e-4,false," ")
 
 --
```

Comment:
why was that empty comment line kept?

Reply:
Take a look at the results with the new tool. I think it does a better job.